config-model/src/main/java/com/yahoo/vespa/model/admin/clustercontroller/ClusterControllerContainerCluster.java

```diff
@@ -28,6 +32,23 @@ protected void doPrepare(DeployState deployState) { }
 
     @Override protected boolean messageBusEnabled() { return false; }
 
+    @Override
+    public void getConfig(QrStartConfig.Builder builder) {
+        int maxHeapSize = featureFlags.clusterControllerMaxHeapSizeInMb();
+        boolean verboseGc = (maxHeapSize < 512);
+        builder.jvm
```

Comment:
Where was this produced earlier on ?

Reply:
ContainerCluster

Reply:
Would prefer a call to `super.getConfig(builder)`, followed by modifications, to avoid duplication.

Reply:
Yup, fixed